### PR TITLE
allow org admins to create apps

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@JayH5 review please

File: mc2/views.py [hunk 1]
```diff
@@ -20,6 +21,14 @@
...
     template_name = 'mc2/home.html'
 
+    def get_context_data(self, *args, **kwargs):
+        context = super(HomepageView, self).get_context_data(*args, **kwargs)
+        active_org = active_organization(self.request)
+        if active_org:
+            context.update(
+                {'is_admin': active_org.has_admin(self.request.user)})
+        return context
+
```
Comment (jerith) [inline mc2/views.py]: What does `get_context_data` do? I'm assuming it's some kind of generic view magic.

Comment (miltontony) [inline mc2/views.py]: @jerith yes, it allows you to add data to the [template context](https://docs.djangoproject.com/en/1.9/ref/class-based-views/mixins-simple/#django.views.generic.base.ContextMixin.get_context_data)

**Comment by jerith** [comment]
:+1:

**Pull request merged**